backport: Rebase to baseline v6.17.13

kbommu opened the pull request:
Signed-off-by: Bommu Krishnaiah <krishnaiah.bommu@intel.com>

kbommu committed:
backport: Rebase to baseline v6.17.13

Signed-off-by: Bommu Krishnaiah <krishnaiah.bommu@intel.com>

ayazahma reviewed (approved):
merge once CI is clean and possible regression or fixed and justified as noty related to current pr

sys-i915-oscijenkins commented:
CI Bug Log - changes from xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55_BAT -> xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1_BAT
====================================================

Summary
-------

  **FAILURE**

  Serious unknown changes coming with xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1_BAT absolutely need to be
  verified manually.
  
  If you think the reported changes have nothing to do with the changes
  introduced in xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1_BAT, please notify your bug team (I915-ci-infra@lists.freedesktop.org) to allow them
  to document this new failure mode, which will reduce false positives in CI.

  

Participating hosts (7 -> 8)
------------------------------

  Additional (1): bat-bmg-1 

Possible new issues
-------------------

  Here are the unknown changes that may have been introduced in xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1_BAT:

### IGT changes ###

#### Possible regressions ####

  * igt@xe_eudebug@attach-debug-metadata:
    - bat-lnl-2:          [PASS][1] -> [ABORT][2]
   [1]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
   [2]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
    - bat-ptl-1:          [PASS][3] -> [ABORT][4]
   [3]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
   [4]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
    - bat-bmg-2:          [PASS][5] -> [ABORT][6]
   [5]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html
   [6]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html

  * igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [FAIL][7]
   [7]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0.html

  
#### Warnings ####

  * igt@xe_eudebug_online@pagefault-write:
    - bat-ptl-2:          [FAIL][8] ([Intel XE#6936]) -> [FAIL][9]
   [8]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html
   [9]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html

  
Known issues
------------

  Here are the changes found in xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1_BAT that come from known issues:

### IGT changes ###

#### Issues hit ####

  * igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-sram@drm_xe_engine_class_render0:
    - bat-lnl-1:          NOTRUN -> [SKIP][10] ([Intel XE#2825]) +3 other tests skip
   [10]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-1/igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-sram@drm_xe_engine_class_render0.html

  * igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-vram@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [SKIP][11] ([Intel XE#6241]) +4 other tests skip
   [11]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-vram@drm_xe_engine_class_compute0.html

  * igt@xe_module_load@load:
    - bat-bmg-1:          NOTRUN -> [ABORT][12] ([Intel XE#1727] / [Intel XE#6930])
   [12]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-bmg-1/igt@xe_module_load@load.html

  
#### Possible fixes ####

  * igt@xe_eudebug@basic-exec-queues-enable:
    - bat-ptl-2:          [FAIL][13] ([Intel XE#6939]) -> [PASS][14]
   [13]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug@basic-exec-queues-enable.html
   [14]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug@basic-exec-queues-enable.html
    - bat-lnl-1:          [FAIL][15] ([Intel XE#6939]) -> [PASS][16]
   [15]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug@basic-exec-queues-enable.html
   [16]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-1/igt@xe_eudebug@basic-exec-queues-enable.html

  * igt@xe_eudebug@discovery-empty:
    - bat-lnl-1:          [FAIL][17] ([Intel XE#6937]) -> [PASS][18] +3 other tests pass
   [17]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug@discovery-empty.html
   [18]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-1/igt@xe_eudebug@discovery-empty.html

  * igt@xe_eudebug@discovery-empty-clients:
    - bat-ptl-2:          [FAIL][19] ([Intel XE#6937]) -> [PASS][20] +3 other tests pass
   [19]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug@discovery-empty-clients.html
   [20]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug@discovery-empty-clients.html

  * igt@xe_eudebug_online@preempt-breakpoint@drm_xe_engine_class_render0:
    - bat-ptl-2:          [FAIL][21] ([Intel XE#6936]) -> [PASS][22] +20 other tests pass
   [21]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug_online@preempt-breakpoint@drm_xe_engine_class_render0.html
   [22]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_eudebug_online@preempt-breakpoint@drm_xe_engine_class_render0.html

  * igt@xe_eudebug_online@reset-with-attention:
    - bat-lnl-1:          [FAIL][23] ([Intel XE#6936]) -> [PASS][24] +26 other tests pass
   [23]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug_online@reset-with-attention.html
   [24]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-1/igt@xe_eudebug_online@reset-with-attention.html

  * igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0:
    - bat-lnl-1:          [FAIL][25] ([Intel XE#6938]) -> [PASS][26] +8 other tests pass
   [25]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0.html
   [26]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-lnl-1/igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0.html

  * igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0:
    - bat-ptl-2:          [FAIL][27] ([Intel XE#6938]) -> [PASS][28] +8 other tests pass
   [27]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0.html
   [28]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1/bat-ptl-2/igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0.html

  
  [Intel XE#1727]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/1727
  [Intel XE#2825]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/2825
  [Intel XE#6241]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6241
  [Intel XE#6930]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6930
  [Intel XE#6936]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6936
  [Intel XE#6937]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6937
  [Intel XE#6938]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6938
  [Intel XE#6939]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6939


Build changes
-------------

  * IGT: igt-xkb-83-c5b9c035a -> igt-xkb-86-165f8a8a8
  * Linux: xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55 -> xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1

  igt-xkb-83-c5b9c035a: c5b9c035a
  igt-xkb-86-165f8a8a8: 165f8a8a8
  xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55: 3b737c674f2daf65c3d16d70dc50ee37634bfa55
  xkb-pr-main-231-418-6001fddc0fb51190083716da70d30143488137a1: 6001fddc0fb51190083716da70d30143488137a1

sys-i915-oscijenkins commented:
CI Bug Log - changes from xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55_BAT -> xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae_BAT
====================================================

Summary
-------

  **FAILURE**

  Serious unknown changes coming with xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae_BAT absolutely need to be
  verified manually.
  
  If you think the reported changes have nothing to do with the changes
  introduced in xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae_BAT, please notify your bug team (I915-ci-infra@lists.freedesktop.org) to allow them
  to document this new failure mode, which will reduce false positives in CI.

  

Participating hosts (7 -> 8)
------------------------------

  Additional (1): bat-bmg-1 

Possible new issues
-------------------

  Here are the unknown changes that may have been introduced in xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae_BAT:

### IGT changes ###

#### Possible regressions ####

  * igt@xe_eudebug@attach-debug-metadata:
    - bat-lnl-2:          [PASS][1] -> [ABORT][2]
   [1]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
   [2]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
    - bat-ptl-1:          [PASS][3] -> [ABORT][4]
   [3]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
   [4]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
    - bat-bmg-2:          [PASS][5] -> [ABORT][6]
   [5]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html
   [6]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html

  * igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [FAIL][7]
   [7]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0.html

  
#### Warnings ####

  * igt@xe_eudebug_online@pagefault-write:
    - bat-ptl-2:          [FAIL][8] ([Intel XE#6936]) -> [FAIL][9]
   [8]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html
   [9]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html

  
Known issues
------------

  Here are the changes found in xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae_BAT that come from known issues:

### IGT changes ###

#### Issues hit ####

  * igt@xe_eudebug_online@writes-caching-sram-bb-sram-target-vram@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [SKIP][10] ([Intel XE#6241])
   [10]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug_online@writes-caching-sram-bb-sram-target-vram@drm_xe_engine_class_compute0.html

  * igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_render0:
    - bat-lnl-1:          NOTRUN -> [SKIP][11] ([Intel XE#2825]) +1 other test skip
   [11]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-1/igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_render0.html

  * igt@xe_module_load@load:
    - bat-bmg-1:          NOTRUN -> [INCOMPLETE][12] ([Intel XE#1727] / [Intel XE#6766] / [Intel XE#6818])
   [12]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-bmg-1/igt@xe_module_load@load.html

  
#### Possible fixes ####

  * igt@xe_eudebug@basic-exec-queues-enable:
    - bat-ptl-2:          [FAIL][13] ([Intel XE#6939]) -> [PASS][14]
   [13]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug@basic-exec-queues-enable.html
   [14]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug@basic-exec-queues-enable.html
    - bat-lnl-1:          [FAIL][15] ([Intel XE#6939]) -> [PASS][16]
   [15]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug@basic-exec-queues-enable.html
   [16]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-1/igt@xe_eudebug@basic-exec-queues-enable.html

  * igt@xe_eudebug@discovery-empty:
    - bat-lnl-1:          [FAIL][17] ([Intel XE#6937]) -> [PASS][18] +3 other tests pass
   [17]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug@discovery-empty.html
   [18]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-1/igt@xe_eudebug@discovery-empty.html

  * igt@xe_eudebug@discovery-empty-clients:
    - bat-ptl-2:          [FAIL][19] ([Intel XE#6937]) -> [PASS][20] +3 other tests pass
   [19]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug@discovery-empty-clients.html
   [20]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug@discovery-empty-clients.html

  * igt@xe_eudebug_online@reset-with-attention:
    - bat-lnl-1:          [FAIL][21] ([Intel XE#6936]) -> [PASS][22] +19 other tests pass
   [21]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_eudebug_online@reset-with-attention.html
   [22]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-1/igt@xe_eudebug_online@reset-with-attention.html

  * igt@xe_eudebug_online@resume-dss@drm_xe_engine_class_render0:
    - bat-ptl-2:          [FAIL][23] ([Intel XE#6936]) -> [PASS][24] +22 other tests pass
   [23]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_eudebug_online@resume-dss@drm_xe_engine_class_render0.html
   [24]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_eudebug_online@resume-dss@drm_xe_engine_class_render0.html

  * igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0:
    - bat-lnl-1:          [FAIL][25] ([Intel XE#6938]) -> [PASS][26] +8 other tests pass
   [25]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-lnl-1/igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0.html
   [26]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-lnl-1/igt@xe_exec_sip_eudebug@breakpoint-waitsip@drm_xe_engine_class_compute0.html

  * igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0:
    - bat-ptl-2:          [FAIL][27] ([Intel XE#6938]) -> [PASS][28] +8 other tests pass
   [27]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55/bat-ptl-2/igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0.html
   [28]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae/bat-ptl-2/igt@xe_exec_sip_eudebug@breakpoint-writesip@drm_xe_engine_class_compute0.html

  
  [Intel XE#1727]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/1727
  [Intel XE#2825]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/2825
  [Intel XE#6241]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6241
  [Intel XE#6766]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6766
  [Intel XE#6818]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6818
  [Intel XE#6936]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6936
  [Intel XE#6937]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6937
  [Intel XE#6938]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6938
  [Intel XE#6939]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6939


Build changes
-------------

  * IGT: igt-xkb-83-c5b9c035a -> igt-xkb-90-165f8a8a8
  * Linux: xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55 -> xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae

  igt-xkb-83-c5b9c035a: c5b9c035a
  igt-xkb-90-165f8a8a8: 165f8a8a8
  xkb-197-3b737c674f2daf65c3d16d70dc50ee37634bfa55: 3b737c674f2daf65c3d16d70dc50ee37634bfa55
  xkb-pr-main-231-420-39c845f8641b5283915645450f8b5198fc50aaae: 39c845f8641b5283915645450f8b5198fc50aaae

kbommu committed:
drm/xe/gt: Fix GT reset handling during worker and unbind

Inline gt_reset() into the GT reset worker to prevent it being called
from incorrect contexts and to clarify the reset flow between the worker
and do_gt_reset(). The previous error return from gt_reset() was also
unused.

Additionally, synchronize GT reset with device unbind by waiting for any
in-progress reset to complete before releasing MMIO mappings. This avoids
MMIO access during unbind, which could otherwise lead to kernel panics.

Add a clarifying comment around the xe_pm_runtime_get()/put() pairing
to make the lifetime rules explicit.

Signed-off-by: Bommu Krishnaiah <krishnaiah.bommu@intel.com>

sys-i915-oscijenkins commented:
CI Bug Log - changes from xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169_BAT -> xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT
====================================================

Summary
-------

  **FAILURE**

  Serious unknown changes coming with xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT absolutely need to be
  verified manually.
  
  If you think the reported changes have nothing to do with the changes
  introduced in xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT, please notify your bug team (I915-ci-infra@lists.freedesktop.org) to allow them
  to document this new failure mode, which will reduce false positives in CI.

  

Participating hosts (7 -> 7)
------------------------------

  No changes in participating hosts

Possible new issues
-------------------

  Here are the unknown changes that may have been introduced in xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT:

### IGT changes ###

#### Possible regressions ####

  * igt@xe_eudebug@attach-debug-metadata:
    - bat-lnl-2:          [PASS][1] -> [ABORT][2]
   [1]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
   [2]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
    - bat-ptl-1:          [PASS][3] -> [ABORT][4]
   [3]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
   [4]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
    - bat-bmg-2:          [PASS][5] -> [ABORT][6]
   [5]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html
   [6]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html

  
Known issues
------------

  Here are the changes found in xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT that come from known issues:

### IGT changes ###

#### Issues hit ####

  * igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_compute0:
    - bat-lnl-1:          NOTRUN -> [SKIP][7] ([Intel XE#2825])
   [7]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-lnl-1/igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_compute0.html

  * igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_render0:
    - bat-ptl-2:          NOTRUN -> [SKIP][8] ([Intel XE#6241]) +2 other tests skip
   [8]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-2/igt@xe_eudebug_online@writes-caching-vram-bb-vram-target-sram@drm_xe_engine_class_render0.html

  
  [Intel XE#2825]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/2825
  [Intel XE#6241]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6241


Build changes
-------------

  * Linux: xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169 -> xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e

  igt-xkb-94-165f8a8a8: 165f8a8a8
  xkb-200-94d58d19fb9f959e8d549e10c389008ec1b08169: 94d58d19fb9f959e8d549e10c389008ec1b08169
  xkb-pr-main-231-427-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e: 319273d29cfe6f43875f40b63d5c8eaf2cc9a94e

sys-i915-oscijenkins commented:
CI Bug Log - changes from xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd_BAT -> xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT
====================================================

Summary
-------

  **FAILURE**

  Serious unknown changes coming with xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT absolutely need to be
  verified manually.
  
  If you think the reported changes have nothing to do with the changes
  introduced in xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT, please notify your bug team (I915-ci-infra@lists.freedesktop.org) to allow them
  to document this new failure mode, which will reduce false positives in CI.

  

Participating hosts (8 -> 8)
------------------------------

  No changes in participating hosts

Possible new issues
-------------------

  Here are the unknown changes that may have been introduced in xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT:

### IGT changes ###

#### Possible regressions ####

  * igt@xe_eudebug@attach-debug-metadata:
    - bat-lnl-2:          [PASS][1] -> [ABORT][2]
   [1]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
   [2]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-lnl-2/igt@xe_eudebug@attach-debug-metadata.html
    - bat-ptl-1:          [PASS][3] -> [ABORT][4]
   [3]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
   [4]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-1/igt@xe_eudebug@attach-debug-metadata.html
    - bat-bmg-2:          [PASS][5] -> [ABORT][6]
   [5]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html
   [6]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-bmg-2/igt@xe_eudebug@attach-debug-metadata.html

  * igt@xe_eudebug_online@pagefault-write:
    - bat-ptl-2:          [PASS][7] -> [FAIL][8]
   [7]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html
   [8]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-2/igt@xe_eudebug_online@pagefault-write.html

  * igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [FAIL][9]
   [9]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-2/igt@xe_eudebug_online@pagefault-write@drm_xe_engine_class_compute0.html

  
Known issues
------------

  Here are the changes found in xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e_BAT that come from known issues:

### IGT changes ###

#### Issues hit ####

  * igt@xe_eudebug@basic-vm-access:
    - bat-bmg-1:          [PASS][10] -> [ABORT][11] ([Intel XE#3970])
   [10]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd/bat-bmg-1/igt@xe_eudebug@basic-vm-access.html
   [11]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-bmg-1/igt@xe_eudebug@basic-vm-access.html

  * igt@xe_eudebug_online@writes-caching-sram-bb-vram-target-sram@drm_xe_engine_class_compute0:
    - bat-lnl-1:          NOTRUN -> [SKIP][12] ([Intel XE#2825]) +4 other tests skip
   [12]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-lnl-1/igt@xe_eudebug_online@writes-caching-sram-bb-vram-target-sram@drm_xe_engine_class_compute0.html

  * igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-vram@drm_xe_engine_class_compute0:
    - bat-ptl-2:          NOTRUN -> [SKIP][13] ([Intel XE#6241]) +4 other tests skip
   [13]: https://intel-gfx-ci.01.org/tree/xe-kmd-backports/xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e/bat-ptl-2/igt@xe_eudebug_online@writes-caching-vram-bb-sram-target-vram@drm_xe_engine_class_compute0.html

  
  [Intel XE#2825]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/2825
  [Intel XE#3970]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/3970
  [Intel XE#6241]: https://gitlab.freedesktop.org/drm/xe/kernel/issues/6241


Build changes
-------------

  * Linux: xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd -> xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e

  igt-xkb-95-a3eb9cd4d: a3eb9cd4d
  xkb-204-b457e22760e3c147eda60d1249e72f1efbd8e8bd: b457e22760e3c147eda60d1249e72f1efbd8e8bd
  xkb-pr-main-231-434-319273d29cfe6f43875f40b63d5c8eaf2cc9a94e: 319273d29cfe6f43875f40b63d5c8eaf2cc9a94e

kbommu commented:
manual testing 
I used same IGT and image(6.17.13-lgci-xe-xkb-pr-main-231-427-31927+) and run the local taskml, I am not able to reproduce the issue https://gtax-ril-fm.intel.com/#/jobs/4373945/task_results/32280636
 
individually also test is passing locally 

[ 1980.019568] [IGT] xe_eudebug: executing
[ 1980.036094] xe 0000:03:00.0: [drm] Tile0: GT0: trying reset from xe_eudebug_enable [xe]
[ 1980.036273] xe 0000:03:00.0: [drm] Tile0: GT0: reset queued
[ 1980.036619] xe 0000:03:00.0: [drm] Tile0: GT0: reset started
[ 1980.050599] xe 0000:03:00.0: [drm] Tile0: GT0: reset done
[ 1980.050614] xe 0000:03:00.0: [drm] Tile0: GT1: trying reset from xe_eudebug_enable [xe]
[ 1980.050766] xe 0000:03:00.0: [drm] Tile0: GT1: reset queued
[ 1980.051327] xe 0000:03:00.0: [drm] Tile0: GT1: reset started
[ 1980.065399] xe 0000:03:00.0: [drm] Tile0: GT1: reset done
[ 1980.069441] [IGT] xe_eudebug: starting subtest attach-debug-metadata
[ 1980.205605] [IGT] xe_eudebug: finished subtest attach-debug-metadata, SUCCESS

igt@xe_eudebug_online@pagefault-write and igt@xe_eudebug_online@pagefault-read test are passing in previous run, it may be noise, ignoring it, I verified locally igt@xe_eudebug_online@pagefault-* passing
